tests/test_provider.py

```diff
@@ -5,3 +5,11 @@ def test_basic(accounts, networks):
 
         assert not receipt.failed
         assert receipt.value == 100
+
+
+def test_receipt(accounts, networks):
```

Comment:
When possible, it is good to do `test_<method_name>`

```suggestion
def test_get_receipt(accounts, networks):
```